modules/cli/src/main/java/org/apache/ignite/cli/call/cliconfig/profile/CliConfigProfileCreateCallInput.java

```diff
@@ -50,35 +53,4 @@ public String getCopyFrom() {
     public boolean isActivate() {
         return activate;
     }
-
-    /**
-     * Builder of {@link CliConfigCreateProfileCallInput}.
-     */
-    public static class CliConfigCreateProfileCallInputBuilder {
```

Comment:
I am quite confused with the fact that you removed the builder only for this input? Why dont remove all builders for CallInput? I think it is ok no to have a builder for the input that has only one field but if the class has more that one field it is quite useful to hava a builder. Bulder helps not to confuse parameter names.